pkgs/applications/editors/neovim/default.nix

```diff
@@ -13,21 +13,21 @@ with stdenv.lib;
 
 let
   neovimLuaEnv = lua.withPackages(ps:
-    (with ps; [ mpack lpeg luabitop ]
+    (with ps; [ compat53 lpeg luabitop luv luv-dev mpack ]
```

Comment:
why do we need this one ? neovim is supposed to run on 5.1 while compat53 provides `Compatibility module providing Lua-5.3-style APIs for Lua 5.2 and 5.1`.

Reply:
cc @gloaming 
I'll try to remove it and build

Reply:
Was not necessary indeed